modules/administration/pages/reports.adoc

```diff
@@ -0,0 +1,63 @@
+. Entitlements: list all organizations, sorted by system or channel entitlements.
+. Patches: list all the patches relevant to the registered systems and sorts patches by severity, as well as the systems that apply to a particular patch.
+. Users: list all registered users and any systems associated with a particular user.
+
+To get the report in CSV format, run this command at the command prompt on the server:
+
+----
+spacewalk-report <report_name>
+----
+
+This table lists the available reports:
+
+
+[[tab.bp.troubleshooting.spacewalk-report]]
+.[command]``spacewalk-report`` Reports
+[cols="1,1,1", options="header"]
+|===
+|Report | Invoked as | Description
+| Channel Packages | [command]``channel-packages`` | All packages in a channel.
+| Channel Report | [command]``channels`` | Detailed report of a given channel.
+| Cloned Channel Report | [command]``cloned-channels`` | Detailed report of cloned channels.
+| Custom Info | [command]``custom-info`` | System custom information.
+| Entitlements | [command]``entitlements`` | All organizations, and their system or channel entitlements.
+| Patches in Channels | [command]``errata-channels`` | All patches in channels.
+| Patches Details | [command]``errata-list`` | All patches that affect registered systems.
+| All patches | [command]``errata-list-all`` | All patches.
+| Patches for Systems | [command]``errata-systems`` | Applicable patches and any registered systems that are affected.
+| Host Guests | [command]``host-guests`` | Host and guests mapping.
+| Inactive Systems | [command]``inactive-systems`` | Inactive systems.
+| System Inventory | [command]``inventory`` | Systems registered to the server, together with hardware and software information.
+| Kickstart Trees | [command]``kickstartable-trees`` | Kickstartable trees.
+| All Upgradable Versions | [command]``packages-updates-all`` | All newer package versions that can be upgraded.
+| Newest Upgradable Version | [command]``packages-updates-newest`` | Newest package versions that can be upgraded.
+| Result of SCAP | [command]``scap-scan`` | Result of OpenSCAP ``sccdf`` evaluations.
+| Result of SCAP | [command]``scap-scan-results`` | Result of OpenSCAP ``sccdf`` evaluations, in a different format.
+| System Data | [command]``splice-export`` | System data needed for splice integration.
+| System Groups | [command]``system-groups`` | System groups.
+| Activation Keys for System Groups | [command]``system-groups-keys`` | Activation keys for system groups.
+| Systems in System Groups | [command]``system-groups-systems`` | Systems in system groups.
+| System Groups Users | [command]``system-groups-users`` | Report of system groups users.
+| Installed Packages | [command]``system-packages-installed`` | Packages installed on systems.
+| Users in the System | [command]``users`` | All users registered to {productname}.
+| Systems administered | [command]``users-systems`` | Systems that individual users can administer.
```

Comment:
Only 24 out of the 51 available reports currently available are listed here.

Source: https://github.com/SUSE/spacewalk/tree/Manager-4.0/reporting/reports/data (every file in this directory is a report)

Do we want to:

1- complete the list,
2- drop the list or
3- only list some here?

Reply:
We probably should list all of them ...